Improve docs metadata (#405)
o1-labs/docs2: docs/zkapps/zkapps-for-ethereum-developers.mdx

---
title: zkApps for Ethereum Developers
hide_title: true

> description: Provides a comprehensive comparison between zkApps and Ethereum smart contracts, discussing differences in language, execution environment, transaction cost, application storage, developer tooling, scaling, and consensus.

Reviewer: ```suggestion
description: How zkApps and Ethereum smart contracts compare, differences in language, execution environment, transaction costs, application storage, developer tooling, scaling, and consensus.
```
